diff --git a/node_modules/react-native-smart-sudoku-grid/Grid.js b/node_modules/react-native-smart-sudoku-grid/Grid.js
index 94eebf9..fc4aafd 100644
--- a/node_modules/react-native-smart-sudoku-grid/Grid.js
+++ b/node_modules/react-native-smart-sudoku-grid/Grid.js
@@ -6,8 +6,7 @@
  */
 
 import React, {
-  Component,
-  PropTypes,
+  Component
 } from 'react'
 import {
   View,
@@ -15,6 +14,8 @@ import {
   Dimensions,
 } from 'react-native'
 
+import { PropTypes } from 'prop-types'
+
 const { width: deviceWidth } = Dimensions.get('window');
 const styles = StyleSheet.create({
   container: {
@@ -32,7 +33,6 @@ export default class SudokuGrid extends Component {
       columnCount: PropTypes.number.isRequired,
       dataSource: PropTypes.array.isRequired,
       renderCell: PropTypes.func.isRequired,
-      style: View.propTypes.style,
   }
 
   // 构造
